### ISSUE TITLE
[CI] Travis CI builds

### ISSUE THREAD

**Issue by TobiTenno** [WFCD/warframe-net#8]
Goals:
- [ ] Build/compile to verify valid code
- [ ] Run tests to validate good reading of the worldstate
- [ ] Publish on good master builds

**Comment by 5herlocked** [comment]
It's working, as far as I can tell.

**Comment by TobiTenno** [comment]
@5herlocked yeah, just don't have all goals done, cause i don't know of tests or publishing methods for this kind of thing
